### PR TITLE
brand added in shop/beauty

### PR DESCRIPTION
**Pull request by amous4822**
#2877 
added perfect brows nyc in shop/beauty

### CODE REVIEW

File: brands/shop/beauty.json [hunk 1]
```diff
@@ -116,5 +116,15 @@
       "name": "Yves Rocher",
       "shop": "beauty"
     }
+  },
+  "shop/beauty|Perfect Brows NYC": {
+    "countryCodes": ["us"],
+    "matchNames": ["Perfect Brows"],
+    "tags": {
+      "beauty": "eyebrows;waxing;eyelash;henna tattoo;facials;men's services;",
```
Comment (tas50) [inline brands/shop/beauty.json]: We probably want to stick with the common known beauty tags here for this to be useful in search / display later

Comment (bhousel) [reply brands/shop/beauty.json]: I agree w @tas50 ..

@amous4822 can you check this list here for commonly used `beauty=*` tags:
https://taginfo.openstreetmap.org/keys/beauty#values

Comment (amous4822) [reply brands/shop/beauty.json]: @tas50 @bhousel I did check. Some of the tags I've added have never been used 😅. I will edit it. Thank you.

Comment (bhousel) [reply brands/shop/beauty.json]: Thanks! looks good 👍

**Pull request merged**